build.gradle

```diff
@@ -6,6 +6,12 @@ plugins {
     id "checkstyle"
 }
 
+ext {
+    if (!project.hasProperty("jacksonVersionForJacksonTest")) {
+        jacksonVersionForJacksonTest = "2.6.7"
```

Comment:
`jacksonVersion` uses `2.6.7.5`. Does this part `2.6.7` explicitly?

Reply:
Ah, good catch! Fixed in: 6f4f237c0e27a3434520ff0a1ca643b499fea7a8